pkg/ddevapp/router_test.go

```diff
@@ -266,3 +269,140 @@ func TestDisableHTTP2(t *testing.T) {
 	assert.Equal("HTTP/1.1 200 OK\r\n", out)
 
 }
+
+// Test the function FindEphemeralPort
+func TestFindEphemeralPort(t *testing.T) {
```

Comment:
I'm not sure, but I think there's a basic misunderstanding here. Both apps should be able to use the same target ports (normally 80 and 443) because ddev-router is routing them correctly. This is the way DDEV normally works.